src/tests/unit/tests/background/actions/inspect-action-creator.test.ts

```diff
@@ -24,7 +25,7 @@ describe('InspectActionCreatorTest', () => {
     beforeAll(() => {
         inspectActionsMock = Mock.ofType(InspectActions, MockBehavior.Strict);
         telemetryEventHandlerMock = Mock.ofType(TelemetryEventHandler, MockBehavior.Strict);
-        browserAdapterMock = Mock.ofType(ChromeAdapter, MockBehavior.Strict);
+        browserAdapterMock = Mock.ofType<BrowserAdapter>(null, MockBehavior.Strict);
```

Comment:
Should `null` be `undefined` to match documented requirement for dynamic mocks [here](https://github.com/florinn/typemoq#dynamic-mocks)? (I didn't see an example of passing `null` as `targetConstructor`)

Reply:
Didn't know about this for the `targerConstructor`. I'll update the ones I change on this PR and keep an eye on other places to update those too.